src/index.html

```diff
@@ -7,12 +7,70 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link
+      href="https://fonts.googleapis.
+      com/css2?family=Poppins:ital,wght@0,100;
+      0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;
+      1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;
+      1,900&family=Roboto:ital,wght@0,100;0,300;0,400;
+      0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;
+      1,900&display=swap" rel="stylesheet">
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
-  <body>
-    <h1>Product cards</h1>
+  <body class="card">
```

Comment:
The `class` attribute here should follow the BEM naming convention. Consider using a more appropriate BEM block name.